packages/e2e-tests/specs/editor/various/inserting-blocks.test.js

```diff
@@ -170,68 +170,6 @@ describe( 'Inserting blocks', () => {
 		).not.toBeNull();
 	} );
 
-	// Check for regression of https://github.com/WordPress/gutenberg/issues/9583
-	it( 'should not allow transfer of focus outside of the block-insertion menu once open', async () => {
```

Comment:
Why are we deleting this test?

Reply:
Because this is testing the side inserter which I removed. (See PR note)